fetch.bs

```diff
@@ -5628,6 +5638,7 @@ interface Request {
   readonly attribute RequestRedirect redirect;
   readonly attribute DOMString integrity;
   readonly attribute boolean keepalive;
+  readonly attribute boolean speculative;
```

Comment:
This would only result in setting the header? You'd still get the response and such?

Reply:
Yeah, that's the current intention. Even though `prefetch` is destined for the next navigation, the current renderer can still get the response (e.g. `onload` fires for it).

Reply:
Well, but this adds it to `fetch()` as a new feature. Was that intended? Or should this only be accessible via `<link>`?

Reply:
I thought it'd be needed to make those headers survive SWs (although that may vary based on where https://github.com/whatwg/fetch/pull/880 ends up)

Reply:
But this would also allow creation. Surviving SW should happen based on copying private state I think.

Reply:
^That sounds right to me. This is similar to the idea that `priority` is a concept that gets [preserved here](https://fetch.spec.whatwg.org/#ref-for-concept-request-priority%E2%91%A1) but is never exposed to the Fetch API.